storage/tokudb/ha_tokudb.cc

```diff
@@ -478,14 +478,7 @@ static inline bool do_ignore_flag_optimization(
 
 #if TOKU_INCLUDE_EXTENDED_KEYS
 static inline uint get_ext_key_parts(const KEY *key) {
```

Comment:
Consider inlining (similar to close_dsmrr() -> ds_mrr.dsmrr_close() change)

Reply:
Yeah, that makes sense, it is only used in one place so no real need for stand alone function.